stubs/pyflakes/pyflakes/checker.pyi

```diff
@@ -169,6 +170,13 @@ else:
     _MatchAs: TypeAlias = Any
     _MatchOr: TypeAlias = Any
 
+if sys.version_info >= (3, 12):
+    _TypeVar: TypeAlias = ast.TypeVar
+    _TypeAlias: TypeAlias = ast.TypeAlias
```

Comment:
Yo dawg, I heard you liked type aliases, so I made a `_TypeAlias` type alias to `TypeAlias`